recipes/sane/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "sane" %}
+{% set version = "1.0.32" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.com/sane-project/backends/uploads/104f09c07d35519cc8e72e604f11643f/sane-backends-{{ version }}.tar.gz
```

Comment:
It it possible to get a preditable URL here rather than having the `104f09c07d35519cc8e72e604f11643f`?